<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryForm</class>
 <widget class="QDialog" name="EntryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gewicht eintragen</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Neuer Eintrag:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="locale">
      <locale language="German" country="Germany"/>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="tabSpacing">
      <number>6</number>
     </property>
     <widget class="QWidget" name="Gewicht">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>582</width>
        <height>91</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Gewicht</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="gewicht">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Gewicht:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Datum: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="datumGewicht">
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="date">
          <date>
           <year>2000</year>
           <month>1</month>
           <day>4</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Umfaenge">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>582</width>
        <height>91</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Umfänge</string>
      </attribute>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>511</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Oberschenkel:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="oberschenkel">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Bauch:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="bauch">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Hüfte:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="huefte">
             <property name="decimals">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Brust:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="brust">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Bizeps:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="bizeps">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Datum:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="datumUmfaenge">
             <property name="displayFormat">
              <string>dd.MM.yyyy</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Ernaehrung">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>582</width>
        <height>91</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Ernährung</string>
      </attribute>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>271</width>
         <height>66</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Protein:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="protein">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Fett:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="fett">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>291</width>
         <height>111</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Gesamtkcal:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Datum:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="gesamtKcal">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Kohlenhydrate:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="kohlenhydrate">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="datumErnaehrung">
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Koerperfettanteil">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>582</width>
        <height>91</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Körperfettanteil</string>
      </attribute>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>581</width>
         <height>81</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>KFA:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="kfa">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Datum:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="datumKFA">
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lustigLol">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Escht wird äbs eigetrooche!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntryForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
